From 7b8426ba73890232cebbef0e7e0775cc17b7a6e9 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Wed, 25 Jul 2012 11:18:01 +0800
Subject: [PATCH 033/602] ENGR00218002 comment out the composite function for
 rndis+adb

RNDIS should work without adb funciton in composite.
To avoid any unnecessary trouble, we should prevent
user to enable USB tethering when ADB is enabled.
Just disable the "rndis,adb" property action to do this.

Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 imx6/etc/init.usb.rc | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/imx6/etc/init.usb.rc b/imx6/etc/init.usb.rc
index a9b832d..c3c8db1 100644
--- a/imx6/etc/init.usb.rc
+++ b/imx6/etc/init.usb.rc
@@ -27,17 +27,6 @@ on property:sys.usb.config=rndis
     write /sys/class/android_usb/android0/enable 1
     setprop sys.usb.state $sys.usb.config
 
-# USB rndis configuration with adb
-on property:sys.usb.config=rndis,adb
-    write /sys/class/android_usb/android0/enable 0
-    write /sys/class/android_usb/android0/idVendor 18d1
-    write /sys/class/android_usb/android0/idProduct 4e24
-    write /sys/class/android_usb/android0/functions $sys.usb.config
-    write /sys/class/android_usb/android0/bDeviceClass 224
-    write /sys/class/android_usb/android0/enable 1
-    start adbd
-    setprop sys.usb.state $sys.usb.config
-
 on property:sys.usb.config=mtp
     write /sys/class/android_usb/android0/enable 0
     write /sys/class/android_usb/android0/idVendor 18d1
-- 
1.8.0

